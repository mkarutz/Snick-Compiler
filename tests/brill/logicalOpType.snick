proc main ()
    int x;
    int y;

    x := 1;
    y := 2;

    write x and y;
end
