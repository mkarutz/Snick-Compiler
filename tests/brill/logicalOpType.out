Failure: Arguments of logical operators must have type bool.
Compilation Failed
